client/src/locales/en.json

```diff
@@ -91,6 +91,10 @@
   "authorization.reset-password.heading": "Reset your password",
   "authorization.reset-password.email-label": "Enter email",
   "authorization.reset-password.button-content": "Reset password",
+  "authorization.actions.no-user-of-email": "There is no user of email {data}.",
+  "authorization.actions.google-user-no-reset-password": "This user: {data} is signed up via Google Account. You can not reset password to Google account here.",
```

Comment:
This is also too much information. We should have some general information for both cases like: "An email with instructions to create a new password has been sent to {email} if it is associated with an EOC account. Your existing password has not been changed." This one is from NVIDIA - check some other services and pick the best. But as You can see they do not say if that email is registered in their DB or not.